sort.py

```diff
@@ -21,5 +23,35 @@ def get_items(self):
         return self._items
 
     def _time(self):
-        self.time = 0
-        return self.time
\ No newline at end of file
+        start_time = time.time()
```

Comment:
Add comments for better readability and understanding.

Reply:
Sure will add in the next release.